varya/assets/sass/blocks/layout-grid/_style.scss

```diff
@@ -1,29 +1,40 @@
 @import 'config';
 
 .wp-block-jetpack-layout-grid {
-	grid-gap: var(--layout-grid--gutter-default);
+	grid-gap: var(--layout-grid--gutter-large) !important;
+	padding-left: var(--layout-grid--gutter-large) !important;
```

Comment:
The addition of this padding causes the content to be misaligned in the editor, from my testing: 

**With padding:**
<img width="600" alt="Screen Shot 2020-04-01 at 10 57 19 AM" src="https://user-images.githubusercontent.com/5375500/78152642-150dae80-7408-11ea-90f4-d26ad62ddd5e.png">

**Without padding:** 
<img width="600" alt="Screen Shot 2020-04-01 at 10 58 54 AM" src="https://user-images.githubusercontent.com/5375500/78152695-28207e80-7408-11ea-8b9f-b56d9236c18d.png">

Not sure if removing this padding causes other issues.

Reply:
> The addition of this padding causes the content to be misaligned in the editor, from my testing:

Ahh, good catch! meant to separate that out, but I’ll do that now and look into the other issue you found as well. 

Thanks!